config-model/src/test/java/com/yahoo/vespa/model/container/ml/ModelsEvaluatorTest.java

```diff
@@ -18,8 +18,11 @@
 public class ModelsEvaluatorTest {
 
     @Test
-    public void testModelsEvaluatorTester() {
+    public void testModelsEvaluator() {
+        // Assumption fails but test passes on Intel macs
+        // Assumption fails and test fails on ARM64
```

Comment:
Confusing comment, so this test always fails?

Reply:
It says "test passes" so I don't know how you summarize it as "always fails".
Anyway, it leaves out linux intel, where (I assume) both the assumption and test succeeds.

Reply:
I understand "assumption fails" as that `assumeTrue` fails, and it was confusing how the test then can pass?

Reply:
Yes, that was surprising to me as well. I guess OnnxEvaluator.isRuntimeAvailable() is wrong.